source/index.ts

```diff
@@ -388,7 +389,16 @@ const predicateOnArray = (method: ArrayMethod, predicate: Predicate, values: unk
 	return method.call(values, predicate);
 };
 
-is.any = (predicate: Predicate, ...values: unknown[]): boolean => predicateOnArray(Array.prototype.some, predicate, values);
+is.any = (predicate: OverloadedPredicate, ...values: unknown[]): boolean => {
+	if (is.array(predicate) === true) {
+		return (predicate as Predicate[]).some((singlePredicate: Predicate) =>
+			predicateOnArray(Array.prototype.some, singlePredicate, values)
+		);
+	}
+
+	return predicateOnArray(Array.prototype.some, predicate as Predicate, values);
+};
```

Comment:
Could be simplified by ensuring an array:
```ts
is.any = (predicate: OverloadedPredicate, ...values: unknown[]): boolean => {
	const predicates = Array.isArray(predicate) ? predicate : [predicate];
	return predicates.some((singlePredicate: Predicate) =>
		predicateOnArray(Array.prototype.some, singlePredicate, values)
	);
};
```

Reply:
This is great - I'll put it in 👍